redaxo/src/core/lib/util/file.php

```diff
@@ -104,6 +104,42 @@ public static function put($file, $content)
         });
     }
 
+    /**
+     * Appends content to a file.
+     *
+     * @param string $file    Path to the file
+     * @param string $content Content for the file
+     * @param string $delimiter delimiter for new Content
+     *
+     * @return bool TRUE on success, FALSE on failure
+     *
+     * @psalm-assert-if-true =non-empty-string $file
+     */
+    public static function append(string $file, string $content, string $delimiter = '')
+    {
+        return rex_timer::measure(__METHOD__, static function () use ($file, $content) {
```

Comment:
```suggestion
        return rex_timer::measure(__METHOD__, static function () use ($file, $content, $delimiter) {
```

Deswegen ist die Variable unten nicht definiert.